app.js

```diff
@@ -217,5 +222,57 @@ messageAllStreaksForChannel = channel => {
 }
 
 messageAllActiveStreaks = msg => {
-  msg.reply(`here are all the active streaks:\n` + bot.buildActiveStreaksMessage())
+  msg.reply(`here are all the active streaks:\n` + bot.buildActiveStreaksMessage(msg.guild.id))
+}
+
+canManageRoles = member => {
+  return member.permissions.bitfield & 0x00000008
+}
+
+getRole = (guild, rolestr) => {
```

Comment:
I think this role management stuff should maybe be in it's own file? Like `services/roles.js`?

Reply:
sure, there is probably a better way of doing this actually, but i didnt see it